.gitlab-ci.yml

```diff
@@ -164,6 +164,7 @@ publish:tests:
   script:
     # Submit coverage from all platforms.
     - for PLATFORM in linux mac; do
+<<<<<<< HEAD
```

Comment:
@danielskinstad This is a sign of a Git conflict incorrectly solved :) Let me know if you want assistance to fix this :+1: